package exceptionHandling;

public class FinnallyBlock {

	public static int getMarks(String sdName) {
		System.out.println("Student gettingmarks :" + sdName);
		if (sdName == "Vijay") 
		{
			try {
				int b = 9 / 0;
			} catch (ArithmeticException e) {
				System.out.println("AE is coming");
				return 100;
			}
			return 75;

		} else if (sdName == "Jay") {
			System.out.println("Jay");
			return 90;
		} else if (sdName == "Raj") {
			System.out.println("Jay");
			return 80;
		}else {
			System.out.println("Student not found");
			return -1;
		}
	}

	public static void main(String[] args) {

		getMarks("Vijay");
//		try {
//			System.out.println("hi");
//			int a = 9 / 0;
//		} catch (ArithmeticException e) {
//			System.out.println("hello");
//			e.printStackTrace();
//		} finally {
//			System.out.println("I'm finally block ");
//		}
	}

}
